### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by BahirHakimy**
# As part of this exercise I have added the following:
- Setup the project using Create React App
- Configured redux using react-redux and redux-toolkit
- Added routing using react-router-dom
- Created Home and Details components
- Added unit tests for redux actions and integration tests for the Home component

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live](https://the-sky.onrender.com/)
+
+## Watch the walkthrough here:
+
+- [Loom Walkthrough](https://www.loom.com/share/77484c8ca17742e6a639bd240fc44ab2)
```
Comment (omarsalem7) [inline README.md]: - [ ] Kindly consider recording a video and make sure the video contains a highlight about a piece of the code. You can consider the following checks when recording the video presentation :
     - [ ] Record a video presenting the features of the project.
     - [ ] The maximum length of the video is 5 minutes.
     - [ ] The video contains a clear and objective description of the project in professional English
     - [ ] The video contains a demo of the project, highlighting all the project features.
     - [ ] The video contains a highlight about a piece of the code that find interesting or is proud of making.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @BahirHakimy 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Your project is deployed properly ✔️
- Connect React and Redux ✔️
- Use store, actions, and reducers in React ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (BahirHakimy) [reply README.md]: @omarsalem7 Thank you very much for your kind review, but I have already recorded the video and I think I have explained all points that you listed, can you please be more specific on which points are missing on that video?

Comment (omarsalem7) [reply README.md]: Hi @BahirHakimy,
I hope you are doing well, but you forget to highlight a piece of code that made you proud of it by sharing a screen with your code in the IDE. you can refer to these requirements at [this link in project documentation](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=you%20should%20also%20highlight%20some%20interesting%20piece%20of%20code%20or%20something%20you%20built%20that%20you%20are%20very%20proud%20of.)
Are you need more explanations?

Comment (BahirHakimy) [reply README.md]: @omarsalem7,
Thank you very much, it's clear now, I will re-record the video,
Thanks for the feedback.

**Review by amejid — APPROVED** [review]
Hi @BahirHakimy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Good presentation :heavy_check_mark: 
- React router used for navigation :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**